CMakeLists.txt

```diff
@@ -1470,23 +1463,13 @@ set(LIBEVENT_INCLUDE_DIRS
     ${EVENT__INCLUDE_DIRS}
     CACHE PATH "Libevent include directories")
 
-configure_file(${PROJECT_SOURCE_DIR}/cmake/LibeventConfigBuildTree.cmake.in
+set(CONFIG_FOR_INSTALL_TREE 0)
+configure_file(${PROJECT_SOURCE_DIR}/cmake/LibeventConfig.cmake.in
                ${PROJECT_BINARY_DIR}/LibeventConfig.cmake
                @ONLY)
```

Comment:
Wrapping this into function would be nice

Reply:
Extracted a macro `gen_package_config()`